st2client/st2client/commands/action.py

```diff
@@ -1379,6 +1384,7 @@ def run(self, args, **kwargs):
     def run_and_print(self, args, **kwargs):
         execution_id = args.id
         output_type = getattr(args, 'output_type', None)
+        include_metadata = args.include_metadata
```

Comment:
The metadata is simply timestamp, right? So should we just say --with-timestamp? Just a thought.

Reply:
It's timestamp + stream / output type.

And we also store some other metadata on the model which we might want to include in the future so perhaps better to keep it as it is.